src/test/java/com/example/universe/simulator/entityservice/unit/services/PlanetServiceTest.java

```diff
@@ -0,0 +1,141 @@
+package com.example.universe.simulator.entityservice.unit.services;
+
+import com.example.universe.simulator.entityservice.entities.Planet;
+import com.example.universe.simulator.entityservice.exception.AppException;
+import com.example.universe.simulator.entityservice.exception.ErrorCodeType;
+import com.example.universe.simulator.entityservice.repositories.PlanetRepository;
+import com.example.universe.simulator.entityservice.services.PlanetService;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.BDDMockito.then;
+import static org.mockito.Mockito.never;
+
+@ExtendWith(MockitoExtension.class)
+class PlanetServiceTest {
+
+    @Mock
+    private PlanetRepository repo;
+
+    @InjectMocks
+    private PlanetService service;
+
+    private static UUID id;
+    private static Planet planet;
+    private static List<Planet> planets;
+
+    @BeforeAll
+    static void setUpContext() {
+        id = UUID.randomUUID();
+        planet = Planet.builder().id(id).name("name").version(0L).build();
+        planets = List.of(planet);
+    }
+
+    @Test
+    @DisplayName("test planetService.get success")
+    void testGet() throws AppException {
+        // arrange
+        given(repo.findById(id)).willReturn(Optional.of(planet));
```

Comment:
Don't use specific parameters in method mocks. Use `ArgumentMatchers`, e.g. `any()`. In your case: `given(repo.findById(any()))`.